MagazineLayout/Public/MagazineLayout.swift

```diff
@@ -755,6 +763,7 @@ public final class MagazineLayout: UICollectionViewLayout {
   override public func invalidateLayout(with context: UICollectionViewLayoutInvalidationContext) {
     guard let context = context as? MagazineLayoutInvalidationContext else {
       assertionFailure("`context` must be an instance of `MagazineLayoutInvalidationContext`")
+      super.invalidateLayout(with: context)
```

Comment:
noticed this was missing a call to super, although this assert / guard should never be hit